src/compiler/build/watch-build.ts

```diff
@@ -53,6 +53,13 @@ export const createWatchBuild = async (
     buildCtx.hasHtmlChanges = hasHtmlChanges(config, buildCtx);
     buildCtx.hasServiceWorkerChanges = hasServiceWorkerChanges(config, buildCtx);
 
+    if (buildCtx.isRebuild) {
+      console.log('🔥:watchBuild:onBuild filesAdded', buildCtx.filesAdded);
```

Comment:
I might suggest also putting the filename or path into the messages we well - that will make it verbose but would permit grepping for all errors in a particular file, for instance

Reply:
That's what I was doing with the string after the first `:` in each log. It maps to the TS file, just in camel cash. We can swap that out tho if it isn't verbose enough. Figured it was enough info for now since we only have logs for this issue in a few spots, and hopefully they won't be around for long 😅

Reply:
ohh gotcha, I didn't read closely enough. that's probably enough for now, I agree 👍